docs/content/direct/multi-wec-aggregated-status.md

```diff
@@ -0,0 +1,171 @@
+# Proposal: Multi-WEC Aggregated Status Controller Enhancement
+
+**Status:** Proposal  
+**Related Issue:** [#2809 – KubeStellar controller logic to map back the status from WECs to READY values in WDSes](https://github.com/kubestellar/kubestellar/issues/2809)
+
+---
+
+## Current Scenario
+
+In the current KubeStellar implementation:
+
+1. **Singleton Enabled (WEC == 1):**  
+   The WDS and WEC workload statuses match correctly when `wantSingletonReportedState: true` is enabled.
+
+2. **Singleton Disabled (WEC > 1):**  
+   For workloads deployed to multiple WECs, the `.status` field in the WDS remains empty, resulting in non-matching WDS and WEC status.
+
+This leads to limited visibility in ArgoCD, where workloads appear as *OutOfSync* or *Unknown*, even though all WECs are healthy.
+
+**Reference:**  
+- [PR #3297](https://github.com/kubestellar/kubestellar/pull/3297)  
+- [Multi-cluster example in docs](https://docs.kubestellar.io/release-0.28.0/direct/example-scenarios/#scenario-1-multi-cluster-workload-deployment-with-kubectl)
+
+---
+
+## Goal
+
+Implement a **multi-WEC aggregated status reporting mechanism** in KubeStellar to reflect combined readiness in the WDS.  
```

Comment:
There already is a feature that reports combined status into the WDS. What is new here is that the combined status goes in the status section of the original workload object.